pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.3"]
+[tools.setuptools_scm]
```

Comment:
I'm new to `pyproject.toml`, but I think this should actually be `[tool.setuptools_scm]`, not `[tools...`. See https://www.python.org/dev/peps/pep-0518/#tool-table and https://github.com/pypa/setuptools_scm/pull/384.